File: iniciante/1019/1019.cpp
// 1019 - Conversão de Tempo - beecrowd
// Iniciante - Nível 1

#include <iostream>

int main() {

    int N;
    std::cin >> N;

    int horas = N / 3600;
    N %= 3600;

    int minutos = N / 60;
    N %= 60;

    int segundos = N;

    std::cout << horas << ":" << minutos << ":" << segundos << '\n';

    return 0;
}

File: iniciante/1019/1019.py
# 1019 - Conversão de Tempo - beecrowd
# Iniciante - Nível 1

N = int(input())

horas = N // 3600
N %= 3600

minutos = N // 60
N %= 60

segundos = N

print(f"{horas}:{minutos}:{segundos}")
